s2a/src/main/java/io/grpc/s2a/handshaker/tokenmanager/SingleTokenFetcher.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2024 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.s2a.handshaker.tokenmanager;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import io.grpc.s2a.handshaker.S2AIdentity;
+import java.util.Optional;
+
+/** Fetches a single access token via an environment variable. */
+public final class SingleTokenFetcher implements TokenFetcher {
+  private static final String ENVIRONMENT_VARIABLE = "S2A_ACCESS_TOKEN";
+
+  /** Set an access token via a flag. */
+  @Parameters(separators = "=")
+  public static class Flags {
+    @Parameter(
+        names = "--s2a_access_token",
+        description = "The access token used to authenticate to S2A.")
+    private static String accessToken = System.getenv(ENVIRONMENT_VARIABLE);
+
+    public synchronized void reset() {
```

Comment:
I'm not sure I understand why 'synchronized' is needed here.

Reply:
This code doesn't exist anymore, as we removed the dependency on Jcommander and replaced  it with a private field & setter